nmmo/core/action.py

```diff
@@ -107,7 +109,6 @@ def edges(cls, config):
   def args(stim, entity, config):
```

Comment:
We may also be able to remove all `args()` methods. In a local branch, I removed all of those, and all tests pass.